new-frontiers-14/frontier-station-14
# Loadouts: fix Lawyer, Janitor slot names, remove Chaplain loadout

## About the PR



Created loadout entries for the Janitor and Lawyer roles.
Originally: Reordered Chaplain loadout categories: mask should be in its correct place.
Edit: Removed Chaplain from Jobs tab in Character Setup window.

## Why / Balance


Consistent loadout screen look compared to other roles.  Originally brought up in #1422 ([comment here](https://github.com/new-frontiers-14/frontier-station-14/pull/1422#pullrequestreview-2079599715)).

## How to test


1. Build and run local server.
2. Open Loadouts screen for Janitor and Lawyer.
3. Check that all expected items are available in the proper tabs.  All role-specific items should be free.
4. There should be no entry for Chaplain under the Jobs tab in the Character Setup window.

## Media


New loadout screens:
![janitor-layout-new](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/d5d6e12f-a905-4a9d-9c07-605ce9b56b7f)
![lawyer-loadout-new](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/2603e865-ced7-4e1d-9d0c-187454b0b5a6)

Old loadout screens:
![janitor-layout-old](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/ae2d7531-9aca-4719-ba12-fdc97be23b3e)
![lawyer-loadout-old](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/0ebc8aad-e028-4c3a-b196-b9f6b3d8c894)


## Breaking changes


**Changelog**



:cl:
- tweak: Lawyer and Janitor loadouts now have consistent tab names and order.
- tweak: Removed Chaplain from Jobs tab in Character Setup.
